go/bin/spec

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+bin/lint
+
+# runs go test on all subdirectories exluding the vendor folder
+# might cause issues with windows users though
```

Comment:
That sounds good. Will also need to exclude the `bin` folder if we are keeping the shared setup